src/sections/SideBar/components/ProfilePic/SideBarProfile.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { makeStyles, Avatar, Typography } from "@material-ui/core";
+import { styles } from "../../../../styles/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: styles.sizes.sidebarDimensions.width,
+    minWidth: styles.sizes.sidebarDimensions.minWidth,
+    maxWidth: styles.sizes.sidebarDimensions.maxWidth,
+    backgroundColor: styles.colors.gray1,
+  },
+  pic: {
+    width: "100%",
+    height: styles.sizes.sidebarDimensions.minWidth,
+  },
+  text: {
+    color: "white",
+    backgroundColor: styles.colors.tBlue,
+    padding: styles.sizes.thickness.thick4,
+    position: "relative",
+    bottom: styles.sizes.specialThick.thick1
+  }
+}));
+
+const ProfilePic = () => {
```

Comment:
This seems to be a generic profile pic we could use in other sections, please move to root level

Reply:
Please add props such as src, alt, .... so that we can specify another image or alt to the Avatar